mne/tests/test_source_estimate.py

```diff
@@ -277,6 +277,10 @@ def test_io_stc():
         assert_array_almost_equal(v1, v2)
     assert_array_almost_equal(stc.tstep, stc2.tstep)
 
+    # Cannot save complex data
+    stc._data = stc._data + 1j
+    assert_raises(RuntimeError, stc.save, op.join(tempdir, "tmp.stc"))
```

Comment:
should also check that `stc._data = stc._data.astype(np.complex128)` passes to hit the other branch